<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>read</class>
 <widget class="QWidget" name="read">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1305</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Расписание</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QRadioButton" name="week1Radio">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1 неделя</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="week2Radio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2 неделя</string>
   </property>
  </widget>
  <widget class="QLabel" name="time3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>331</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>395</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="time4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>421</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>491</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="time5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>517</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>581</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="time6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>607</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>218</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>305</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="time2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>244</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="para1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>124</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="time1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>1221</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>135</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="facultLabel">
        <property name="text">
         <string>Факультет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="facultBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="groupLabel">
        <property name="text">
         <string>Группа</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="groupBox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="abtButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>1124</x>
     <y>11</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>61</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>tableWidget</zorder>
  <zorder>week1Radio</zorder>
  <zorder>week2Radio</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>time3</zorder>
  <zorder>para4</zorder>
  <zorder>time4</zorder>
  <zorder>para5</zorder>
  <zorder>time5</zorder>
  <zorder>para6</zorder>
  <zorder>time6</zorder>
  <zorder>para2</zorder>
  <zorder>para3</zorder>
  <zorder>time2</zorder>
  <zorder>para1</zorder>
  <zorder>time1</zorder>
  <zorder>abtButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
